Doc/library/sys.rst

```diff
@@ -11,6 +11,28 @@ interpreter and to functions that interact strongly with the interpreter. It is
 always available. Unless explicitly noted otherwise, all variables are read-only.
 
 
+.. data:: abi_features
+
+   A frozen set of ABI features.
+   Currently supported features are:
+
+   ``free-threading``
+      The Python interpreter is free-threading. On Unix systems, this corresponds
+      to ``"t" in sys.abiflags``.
+
+   ``debug``
+      The Python interpreter has debugging capabilities. On Unix systems, this
+      corresponds to ``"d" in sys.abiflags``.
+
+   ``32-bit`` or ``64-bit``
+      The bitness of the interpreter, that is, whether it is a 32-bit or 64-bit
+      build.
+
+   .. versionadded:: 3.14
+
+   .. availability:: Unix, Windows
```

Comment:
No need for an `.. availability` directive, if it's available everywhere. Also, Unix & Windows is not an exhaustive list.